core/themes/seven/css/style.css

```
@@ -652,85 +652,89 @@ tr.selected td {
  *   to apply all padding to the inner .fieldset-wrapper instead.
  */
 fieldset {
+  position: relative;
   max-width: 100%;
   margin: 1em 0;
   padding: 0;
+  border-radius: 4px;
   background-color: #fff;
-  border: none;
+  border: 2px solid #EAEAEA;
```

As per [CSS coding standards](https://api.backdropcms.org/css-standards#format), hex values should be lowercase:

```suggestion
  border: 2px solid #eaeaea;
```

---

Will do so. :+1: